cmd/gardenlet/app/app.go

```diff
@@ -107,8 +107,8 @@ func run(ctx context.Context, cancel context.CancelFunc, log logr.Logger, cfg *g
 		cfg.SeedClientConnection.Kubeconfig = kubeconfig
 	}
 
-	log.Info("Getting rest config for seed")
-	seedRESTConfig, err := kubernetes.RESTConfigFromClientConnectionConfiguration(&cfg.SeedClientConnection.ClientConnectionConfiguration, nil)
+	log.Info("Getting rest config for runtime cluster")
```

Comment:
shouldn't we just call it cluster? why runtime cluster?
```suggestion
	log.Info("Getting rest config for cluster")
```

Reply:
The `gardenlet` also creates a `cluster.Cluster` (with a separate `rest.Config`) for the garden cluster. Hence, I prefer to keep this log message somewhat specific. "seed" is not accurate anymore, and typically we use "runtime" as term.

Reply:
I also can't think of a better term. 🥲